x/programs/runtime/import_state.go

```diff
@@ -95,6 +99,22 @@ func NewStateAccessModule() *ImportModule {
 
 				return bytes, nil
 			})},
+			"delete_many": {FuelCost: putManyCost, Function: FunctionNoOutput(func(callInfo *CallInfo, input []byte) error {
+				var parsedInput []keyInput
```

Comment:
This should be able to just be [][]byte.

Reply:
true, no reason to define a struct

Reply:
thank you!